libcxx/include/ios

```diff
@@ -250,6 +250,8 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 typedef ptrdiff_t streamsize;
 
+struct _LIBCPP_HIDE_FROM_ABI uninitialized_ios{};
```

Comment:
This type should have an __ugly name. And perhaps it should have an explicit default constructor.
```suggestion
struct __uninitialized_ios_tag {
  explicit __uninitialized_ios_tag() = default;
};
```